components/openindiana/nvidia-select/files/nvidia-select

```diff
@@ -0,0 +1,415 @@
+#! /bin/sh
+
+#
+# Select one of the installed mediatable NVidia drivers
+# as the one to run for your current system under BASEDIR.
+# This should be executed as root (or sufficiently privileged user).
+#
+
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+# Copyright 2017, Jim Klimov
+# Portions Copyright 2003-2005 Sun Microsystems, Inc.  All rights reserved.
```

Comment:
I'm not sure about 'Portions Copyright' string. Do we have precedents?
I'd just put Sun copyright on top, if this file has Sun heritage.

Reply:
Gotta remember now, not sure I can after so long :)

I'd guess this started by looking at `ogl-select` and at least modelling after the idea.

Reply:
Looking at current `ogl-select`, they are very dissimilar. So I must have started from something else back then :)

As for precedents, we have a few:

````
jim@jimoi:~/shared/oi-userland-tmp/components$ git grep -i 'portions copyright' | grep -v yyyy | wc -l
32
jim@jimoi:~/shared/oi-userland-tmp/components$ git grep -i 'portions copyright' | wc -l
1132
````